source/admin_manual/known_issues/mbox_problems.rst

```diff
@@ -0,0 +1,158 @@
+   the same index file for multiple different mboxes!
+
+   -  This could happen if you let Dovecot do mailbox autodetection and
+      it sometimes uses ``/var/mail/%u`` (when it exists) and other
+      times ``~/mail/inbox``. Use an explicit
+      :dovecot_core:ref:`mail_location`
+      setting to make sure the same INBOX is used.
+
+   -  Another possibility is that you're sharing index files between
+      multiple users. Each user must have their own home directory.
+
+It's possible that broken X-UID headers in mails and
+:dovecot_core:ref:`mbox_lazy_writes=yes <mbox_lazy_writes>` combination has some bugs.
+If you're able to reproduce such an error, please let us know how.
+
+UIDVALIDITY changes
+~~~~~~~~~~~~~~~~~~~
+
+UIDVALIDITY is stored in X-IMAPbase: or X-IMAP: header of the first
+message in mbox file. This is done by both Dovecot and UW-IMAP (and
+Pine). It's also stored in ``dovecot.index`` file. It shouldn't normally
+change, because if it does it means that client has to download all the
+messages for the mailbox again.
+
+If the UIDVALIDITY in mbox file doesn't match the one in
+``dovecot.index`` file, Dovecot logs an error:
+
+::
+
+   UIDVALIDITY changed (1100532544 -> 1178155834) in mbox file /home/user/mail/mailbox
+
+This can happen when the following happens:
+
+1. Dovecot accesses the mailbox saving the current UIDVALIDITY to
+   ``dovecot.index`` file.
+
+2. The UIDVALIDITY gets lost from the mbox file
+
+   -  X-IMAP: or X-IMAPbase: header gets lost because something else
+      than Dovecot or UW-IMAP deletes the first message
+
+   -  The whole file gets truncated
+
+   -  Something else than Dovecot deletes or renames the mbox
+
+3. The mailbox is accessed (or created if necessary) by UW-IMAP or Pine.
+   It notices that the mailbox is missing UIDVALIDITY, so it assigns a
+   new UIDVALIDITY and writes the X-IMAPbase: or X-IMAP: header.
+
+   -  Also Dovecot that's configured to not use index files behaves the
+      same.
+
+4. Dovecot accesses again the mailbox. UIDVALIDITY in the mbox file's
+   header doesn't match the one in ``dovecot.index`` file. It logs an
+   error and updates the UIDVALIDITY in the index file to the new one.
+
+Crashes
+-------
+
+Dovecot's mbox code is a bit fragile because of the way it works.
+However instead of just corrupting the mbox file, it usually
+assert-crashes whenever it notices an inconsistency. You may see crashes
+such as:
+
+::
+
+   Panic: mbox /home/user/mail/mailbox: seq=2 uid=45 uid_broken=0 originally needed 12 bytes, now needs 27 bytes
+
+This is a bit difficult problem to fix. Usually this crash has been
+related to Dovecot rewriting some headers that were broken. If you see
+these crashes, it would really help if you were able to reproduce the
+crash.
+
+If you have such a mailbox which crashes every time when it's tried to
+be opened, please put the mbox through `mbox
+anonymizer <https://dovecot.org/tools/mbox-anonymize.pl>`__ and send it,
```

Comment:
The link needs to be updated to "https://github.com/dovecot/tools/blob/main/mbox-anonymize.pl".